spec/services/search_indexer_spec.rb

```diff
@@ -158,7 +158,7 @@ def scrub(html, strip_diacritics: false)
 
       post.rebake!
       post.reload
-      topic = post.topic
+      post.topic
```

Comment:
I think this line is safe to remove since it isn't being used.

Reply:
probably, we can try that.... I was just fixing complaints from my linter.